app/services/symphony_client.rb

```diff
@@ -41,24 +41,28 @@ def session_token
     end
   end
 
-  # Symphony API requires some loooong strings, disabling LineLength for this method
-  # rubocop:disable Metrics/LineLength
+  ITEM_RESOURCES = 'item{*,bib{title,author},call{sortCallNumber,dispCallNumber}}'
+
+  PATRON_LINKED_RESOURCES = [
+    "holdRecordList{*,#{ITEM_RESOURCES}}",
+    "circRecordList{*,circulationRule{loanPeriod{periodType{key}}},#{ITEM_RESOURCES}}",
```

Comment:
I think that `renewFromPeriod` might have gotten dropped from this. I think this is needed to determine if an item is renewable and is still needed.

Reply:
Good catch; I'll add that back.